src/applications/personalization/account/components/TermsAndConditions.jsx

```diff
@@ -5,8 +5,7 @@ import isBrandConsolidationEnabled from '../../../../platform/brand-consolidatio
 const propertyName = isBrandConsolidationEnabled() ? 'VA.gov' : 'Vets.gov';
 
 export default function TermsAndConditions({ mhvAccount }) {
-  const termsConditionsUrl =
-    '/health-care/medical-information-terms-conditions';
+  const termsConditionsUrl = '/health-care/';
```

Comment:
Hm, this is the right redirection, but doesn't make sense in context. We'll have to do something else here, because right now that link on Staging doesn't make sense since it just redirects you to `/health-care/`.

![image](https://user-images.githubusercontent.com/1915775/49660312-43e26300-fa14-11e8-868b-bfaf1db038d9.png)

Reply:
Per this PR, https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15620, we are un-retiring this page so that the Terms and Conditions app remains accessible. This app is necessary at this time.